code/modules/events/spacevine.dm

```diff
@@ -491,19 +507,15 @@
 	vines += SV
 	SV.master = src
 	if(muts && muts.len)
-		SV.mutations |= muts
+		for(var/datum/spacevine_mutation/M in muts)
+			M.add(SV)
+		return
```

Comment:
If you specified the list of mutations new one needs to recieve you sure as hell dont want any random mutation to pop up